engines/director/detection_tables.h

```diff
@@ -402,6 +402,8 @@ static const PlainGameDescriptor directorGames[] = {
 	{ "rhem1",				"Rhem" },
 	{ "rhlava",				"Rescue Heroes: Lava Landslide" },
 	{ "rhmeteor",			"Rescue Heroes: Meteor Madness" },
+	{ "robinson",			"Robinson Crusoe" },
+	{ "robinson-makingof",	"Robinson Crusoe: Making-of" },
```

Comment:
I think the same entry can be used for the "making of", as in `jman` or `mirage`

Reply:
This might be problematic in this case, because `robinson-makingof` is using a different Director version, thus we might have to implement specific quirks for this.